R/bnma_analysis.R

```diff
@@ -8,11 +8,11 @@
 #'  - 'ArmLevel' = Data frame containing 'Study', 'Treat', 'N', 'Outcomes', and (for outcome_type="Continuous") 'SD'
 #'  - 'Treat.order' = Vector of (unique) treatments, with the reference treatment first
 FormatForBnma <- function(br_data, treatment_ids, outcome_type, ref){
-  if(FindDataShape(br_data) == "wide"){
+  if (FindDataShape(br_data) == "wide"){
     br_data2 <- as.data.frame(WideToLong(br_data, outcome_type=outcome_type))
-  } else if(FindDataShape(br_data) == "long"){
+  } else if (FindDataShape(br_data) == "long"){
     br_data2 <- br_data
-  } else {paste0("data_format has to be 'wide' or 'long'")}
+  } else {stop("data_format has to be 'wide' or 'long'")}
```

Comment:
Can you break this into an indented block please?

Reply:
Done.